doc/csi.md

```diff
@@ -7,10 +7,10 @@ document.
 Basic workflow starting from registration is as follows:
 
 1. csi-node-driver-registrar retrieves information about csi plugin (mayastor) using csi identity service.
-2. csi-node-driver-registrar registers csi plugin with kubelet passing plugin's csi endpoint as parameter.
-3. kubelet uses csi identity and node services to retrieve information about the plugin (including plugin's ID string).
-4. kubelet creates a custom resource (CR) "csi node info" for the CSI plugin.
-5. kubelet issues requests to publish/unpublish and stage/unstage volume to the CSI plugin when mounting the volume.
+1. csi-node-driver-registrar registers csi plugin with kubelet passing plugin's csi endpoint as parameter.
+1. kubelet uses csi identity and node services to retrieve information about the plugin (including plugin's ID string).
+1. kubelet creates a custom resource (CR) "csi node info" for the CSI plugin.
+1. kubelet issues requests to publish/unpublish and stage/unstage volume to the CSI plugin when mounting the volume.
```

Comment:
In MD, the numbers are displayed correctly when written as such. They resolve to 1. 2. 3
. Ext.

It's generally considered better when numbering to leave them as all 1. So that if you change (add or remove) you don't need to readjust the numbering. Technically, MD ignores what the numbers actually are besides the starting one. So:

1
5
10

Are actually rendered

```md
1. One
5. Five
10. Ten
```


Happy to change it back to hard numbered though if you'd prefer.

Reply:
Ok - I didn't realise that was the case. Thanks for clarifying. In that case ignore the other similar comments that I have made.